### PR TITLE
Use env variables

### LINKED ISSUE #29: Use env variables

### PR DESCRIPTION
**Pull request by manu-4216**
closes #29

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -4,8 +4,10 @@ const app = express();
 const users = require("./routes/users");
 const projects = require("./routes/projects");
 const bodyParser = require("body-parser");
-const { dbPassword } = require("./config");
 const port = 4000;
+require("dotenv").config();
+const dbPassword = process.env.dbPassword;
+
 console.log(dbPassword);
```
Comment (fmtabbara) [inline server/server.js]: Maybe remove the console.log() ?

Comment (fmtabbara) [inline server/server.js]: Maybe remove the console.log() ?

Comment (manu-4216) [reply server/server.js]: Done

**Review by fmtabbara — COMMENTED** [review]
Good to go

**Review by fmtabbara — APPROVED** [review]

**Pull request merged**